frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentBlock.tsx

```diff
@@ -162,16 +163,13 @@ export const StripePaymentBlock = ({
           <Text textStyle="h3" textColor="primary.500" mb="1rem">
             Payment
           </Text>
-          <PaymentItemDetailsBlock
+          <FixedPaymentItemDetailsBlock
```

Comment:
Was wondering if we can document this better, agreed it makes sense to re-use the fixed payment block for both variable payment and fixed payment in payment page, as by that time the payment amount is 'fixed'. 

But cause at first I got confused at this line (wondering why there wasn't a separate one for variable 😅)